backend/business-services/egov-hrms/README.md

```
@@ -0,0 +1,112 @@
+
+**a) Assignments:** Every employee is assigned a list of assignments, every assignment is a designation provided to that employee for a given period of time. These designations are mapped to departments. This also includes marking the employee as HOD for that dept if needed. Employee can also provide information on who does he report to.
+   
+   - **Constraints:**
+        1. For a given period of time an employee shouldn't have more than one assignments.
+        2. The department and designation part of the employee must be configured in the system.
+        3. Details of assignment once entered in the system cannot be deleted.
+        4. An employee cannot have more than one active assignment.
+
+**b) Jurisdictions:** A jurisdiction is a area of power for any employee. It can be a zone, ward, block, city, state or the country. Currently a jurisdiction is defined as combination of Hierarchy type, Boundary Type and the actual Boundary. However, in the current system we are not validating these jurisdictions. This is being collected only for the sake of data.
+   
+   - **Constraints:**
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)